CMakeLists.txt

```diff
@@ -203,8 +203,6 @@ foreach(prec ${precisions})
       TARGET ${LIBNAME}_${prec}
       SOURCES ${prec_srcs} $<TARGET_OBJECTS:${LIBNAME}>
       DEFINITIONS $<$<NOT:${fiat_FOUND}>:${FIELD_API_DEFINITIONS}>
-      INTERFACE_LIBS
-         $<${HAVE_ACC}:OpenACC::OpenACC_Fortran>
```

Comment:
Do we really not need to link to OpenACC in all cases? I suspect we are getting away with that on NVHPC because the relevant flags added below may automatically imply that behaviour, but CCE or GNU may be handling this differently for example?

Reply:
`ecbuild_add_library` doesn't even have an INTERFACE_LIBS option in the API. So INTERFACE_LIBS was in fact being interpreted as a compiler definition. We put that in to avoid F-API exporting openacc compiler flags, and so we were accidentally getting the right behaviour. In any case, a true interface library would indeed export compiler flags too, not just linker flags, so explicitly exporting link flags is the right way to do it.

Reply:
Thanks, I wasn't aware of this. Looks all good then to me, sorry for the noise!